04_verify.sh

```diff
@@ -305,6 +305,11 @@ for container in ${EXPTD_CONTAINERS}; do
   iterate check_container "$container"
 done
 
+# Add delay before checking the ironic endpoint in Centos(minikube) test
+# This will provide extra time for ironic pod to be in running state
+if [[ "${SKIP_APPLY_BMH:-false}" == "true" ]] && [[ "${EPHEMERAL_CLUSTER}" == "minikube" ]]; then
+  sleep 300
```

Comment:
Can we check the ironic pod status with a timeout instead of `sleep 300`? So we don't have to always wait for 300 and progress further if the pod is ready earlier.